### ISSUE TITLE
New W11 update (KB5023778) makes the File Explorer crash whenever it tries to show the Start Menu

### ISSUE THREAD

**Issue by Tiagoquix** [valinet/ExplorerPatcher#1211]
New update link: https://support.microsoft.com/en-us/topic/march-28-2023-kb5023778-os-build-22621-1485-preview-d490bb51-492e-410c-871f-50ad01b0f765

ExplorerPatcher and File Explorer works fine, except the Start Menu, which doesn't open and crashes when I try to open it.

**Comment by MystiqLeaf** [comment]
how did you get to the properties, because even right clicking on taskbar causes it to restart

**Comment by darthbiggious** [comment]
> Just disabled the "Show Search button" option in the application properties and restarted explorer.exe a few times. After that, everything was back to normal.


not sure that's the problem, show search button was still disabled on mine and the start menu still won't open

**Comment by Gxtod** [comment]
> > Just disabled the "Show Search button" option in the application properties and restarted explorer.exe a few times. After that, everything was back to normal.
> 
> not sure that's the problem, show search button was still disabled on mine and the start menu still won't open

I confirm the problem still persists... (

**Comment by darthbiggious** [comment]
everything seems to be backing working again here after an icon update this morning

**Comment by vittoriop77** [comment]
> > > Just disabled the "Show Search button" option in the application properties and restarted explorer.exe a few times. After that, everything was back to normal.
> > 
> > 
> > not sure that's the problem, show search button was still disabled on mine and the start menu still won't open
> 
> I confirm the problem still persists... (

Me too

**Comment by ohiosumi** [comment]
![image](https://user-images.githubusercontent.com/115827857/228768704-9cf91e43-e28f-4dab-9aec-b2b8a6286cf1.png)
it throws this error dialog

**Comment by Nelo-cool** [comment]
I can confirm)  it can't show the start menu. Uninstall this application, until I have to uninstall the app  ExplorerPatcher

**Comment by gangefors** [comment]
I've tried a lot of different settings in regards to the task bar but I get crashes no matter what I use. But it might be somehow related to search anyway. Because after the Patcher has crashed and I've clicked the dismiss box, I can open the start menu. But as soon as I click in the search box in the open Start menu it crashes again.
Uninstalling seems to be the only solution to get the start menu working again.
I hope that someone can find the root cause and fix it.

**Comment by darthbiggious** [comment]
it was sort of working again after a symbols update, but 90% of pinned apps & about 50% of the stuff on the all apps list were missing from windows 10 mode and couldn't be re-pinned. Reset my settings to default incase I'd messed something up & it  started crashing again, but at the moment as long as I leave the start menu set to windows 11, the taskbar and everything is working as expected.

**Comment by RolandWH** [comment]
Was broken for me at first but now seems to work fine after being allowed to download new symbols.

**Comment by ChristianGruen** [comment]
> Was broken for me too at first but now seems to work fine after being allowed to download new symbols.

Same for me: I don’t encounter any crashes anymore.
Thanks, in general, for the great software!

**Comment by gangefors** [comment]
I just downloaded the 22621.1413.54.5 release again and did a fresh install from the .exe. No dice! Taskbar crashes on right click and on start menu click. There is something very incompatible with the current state of an updated Windows 11 and latest ExplorerPatcher.

**Comment by RolandWH** [comment]
> I just downloaded the 22621.1413.54.5 release again and did a fresh install from the .exe. No dice! Taskbar crashes on right click and on start menu click. There is something very incompatible with the current state of an updated Windows 11 (22621.1485) and latest ExplorerPatcher.

@gangefors Did you try launching ExplorerPatcher and waiting a minute or two for it to download new symbols without inducing a crash by interacting with the taskbar? This is what fixed it for me.

**Comment by gangefors** [comment]
> Did you try launching ExplorerPatcher and waiting a minute or two for it to download new symbols without inducing a crash by interacting with the taskbar? This is what fixed it for me.

So I've figured out why my reinstallation didn't work. For some reason the _Enable symbols download_ was disabled. But since I couldn't even enter the Start Menu or right click the task bar to get to the Properties window it was close to impossible for me to know that this was the root cause.

To solve it I opened an Explorer window and navigated to the Start Menu manually to find the shortcut to open the Properties window. I.e. navigating to C:\ProgramData\Microsoft\Windows\Start Menu\Programs\ExplorerPatcher or just run `C:\Windows\System32\rundll32.exe "C:\Program Files\ExplorerPatcher\ExplorerPatcher.amd64.dll",ZZGUI` using Win+R. After opening the Properties window IO navigated to Advanced and marked Enable symbols download. Waited until the notification showed up that new symbols were downloaded, and last click Restart File Explorer in the bottom of the window.

This is how I solved my crashing Taskbar and Start Menu. Hope this helps someone else. Thanks @RolandWH for getting me on the correct path.
